crates/solidity/testing/snapshots/cst_output/Identifier/keyword_abstract/generated/0.6.0.yml

```diff
@@ -0,0 +1,16 @@
+# This file is generated automatically by infrastructure scripts. Please don't edit by hand.
+
+Source: >
+  1  │ abstract                                                                         │ 0..8
+
+Errors: # 1 total
+  - >
+    Error: Expected Identifier.
```

Comment:
Not sure I understand why was it looking for an `Identifier` here?

Reply:
Because that is the production we are testing.

Reply:
Oh, totally misread that :D